<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RFID Reader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>457</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuConnect_Device">
    <property name="title">
     <string>Connect/Disconnect</string>
    </property>
    <addaction name="actionConnect_Device"/>
    <addaction name="actionDisconnect_Device"/>
   </widget>
   <addaction name="menuConnect_Device"/>
   <addaction name="menuExit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>&amp;Config</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
  </action>
  <action name="actionConnect_Device">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect_Device">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
